src/test/java/executor/service/services/ProxySourcesClientTest.java

```diff
@@ -9,10 +9,11 @@
 import org.junit.jupiter.api.Test;
 import java.io.*;
 import java.util.ArrayList;
+import java.util.List;
 
 class ProxySourcesClientTest {
```

Comment:
let's use common code style
```suggestion
public class ProxySourcesClientTest {
```

Reply:
done